mongoengine/python_support.py

```diff
@@ -4,8 +4,11 @@
 import pymongo
 
 
+IS_PYMONGO_27 = False
```

Comment:
I think we could be more explicit with this test. For example PyMongo v1.7, v2.6, etc. would make `IS_PYMONGO_27` true right now (no matter if they actually exist/are used in the wild).

How about:
```
IS_PYMONGO_27 = pymongo.version_tuple[0] == 2 and pymongo.version_tuple[1] == 7
```